CONTRIBUTING.md

```diff
@@ -0,0 +1,16 @@
+## Making changes to CitGM
+#### There are a few basic requirements for making changes to CitGM
+
+* Create an issue prior to submitting PR's as this can help speed up the process
+* Include tests for your code wherever possible
+* Include any documentation or manual changes where necessary
+* Ensure that `npm test` passes before submitting the PR
+* Please squash each logical change into one single commit
+* Follow the nodejs commit guidelines shown [here](https://github.com/nodejs/node/blob/master/CONTRIBUTING.md#step-3-commit)
```

Comment:
we have not been following these commit guidelines. Do we want to start doing so? This feels like it should likely be it's own issue to discuss